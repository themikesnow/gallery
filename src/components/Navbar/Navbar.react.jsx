import React from 'react';

const MyNavBar = () => (
  <div className="navbar">
    <div className="content">
      <div className="brand">
        Gallery
      </div>
    </div>
  </div>
);

export default MyNavBar;
