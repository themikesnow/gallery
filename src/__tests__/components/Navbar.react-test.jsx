import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { expect as expectChai } from 'chai';

import Navbar from '../../components/Navbar/Navbar.react';

describe('<Navbar />', () => {
  it('Match Snapshot', () => {
    const component = renderer.create(<Navbar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render classes', () => {
    const wrapper = shallow(<Navbar />);
    expectChai(wrapper.find('.navbar')).to.have.length(1);
    expectChai(wrapper.find('.content')).to.have.length(1);
    expectChai(wrapper.find('.brand')).to.have.length(1);
  });
});
