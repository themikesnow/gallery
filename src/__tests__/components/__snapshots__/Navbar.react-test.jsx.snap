// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<Navbar /> Match Snapshot 1'] = `
<div
  className="navbar"
>
  <div
    className="content"
  >
    <div
      className="brand"
    >
      Gallery
    </div>
  </div>
</div>
`;
